testnets/autheotestnet/assetlist.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "autheotestnet",
+  "assets": [
+    {
+      "description": "Native token of Autheo Chain",
+      "denom_units": [
+        {
+          "denom": "aauth",
+          "exponent": 0
+        },
+        {
+          "denom": "aauth",
+          "exponent": 5
+        }
+      ],
+      "base": "aauth",
+      "name": "Autheo Chain",
+      "display": "autheo",
+      "symbol": "THEO",
+      "logo_URIs": {
```

Comment:
The convention in this repository is to keep the file in this registry itself. Because you technically won't be able to commit the file to main branch until merged, just swap the branch to main.